src/fsqueue.py

```diff
@@ -97,11 +107,16 @@ def main(parsedargs) :
         jobidlist = []
         for j in args.jobs.split(",") :
             jobidlist.append(fjob.id_parse(j))
+            verbose_cmd += j + " "
         mylist = fjob.JobList(myhandle,user=user,ids=jobidlist)
     if args.nodelist:
         myjoblist = filter_byhostlist(mylist, args.nodelist)
+        if args.verbose == True :
+            print("#warning: hostlist filtering not possible with flux-jobs CLI")
```

Comment:
Hi @gonsie! Sweet this is looking good! I'm not sure how we want to standardize warning messages to the user but I might suggest modifying your `#warning: ...` statements to `WARNING: ...` to match consistency within `fsqueue.py`.

Reply:
I think it makes sense to use `WARNING: ...` with anything that we expect to be read by people, but I can see a case for `#warning: ...` for things that you expect to be read by programs. It doesn't look like I've been entirely consistent in other places in the wrappers, but I'd be good with standardizing on `WARNING:`.